collect_app/src/test/java/org/odk/collect/android/gdrive/SheetsHelperTest.java

```diff
@@ -99,6 +104,42 @@ public void getSpreadsheetTest() throws IOException {
         assertBatchUpdateCalled(1);
     }
 
+    @Test
+    public void whenNewSpreadsheetDetected_shouldBatchUpdateBeCalled() throws IOException {
+        doReturn(true).when(sheetsHelper).isNewSpreadsheet("spreadsheet_id", "Sheet1");
```

Comment:
It looks like this test file uses a spy to mock out parts of the object under test which y'all know I'm not a fan of. This is something we'd need to take some time to untangle (like headphones in a pocket) however as it's an existing problem. I think lets go with this for now!

By the way I do think these new tests have been structured really nicely. There's clear separation between what is testing `updateSpreadsheetLocaleForNewSpreadsheet` and what is testing `isNewSpreadsheet`. That'll make it a lot easier to pull apart later.